test/github.test.js

```diff
@@ -0,0 +1,86 @@
+const gh = require('../src/github-url');
+
+/**
+ * Test to validate fetching data when passing a user URL.
+ * It uses the Seneca-CDOT URL
+ */
+test('test fetching data for a valid user URL', async () => {
+  const validUserUrl = 'https://github.com/Seneca-CDOT';
```

Comment:
Testing against the live API has the potential to fail (network down, API down, throttling, etc).  Often people will mock network APIs in tests, and give back simulated responses.  Here's an example you can look at https://github.com/humphd/learn-travis/blob/master/test/app.mocked.test.js.  In a follow up, we can file an issue to use mock responses here.